raw_install_python.yml

```diff
@@ -50,8 +50,25 @@
       when: stat_apt.rc == 0
 
     - name: install python for opensuse
-      raw: zypper -n install python-base
+      raw: zypper -n install python-base python-xml
       register: result
       until: result is succeeded
       when: stat_zypper.rc == 0
-  when: not True in (systempython.results | selectattr('stat', 'defined') | map(attribute='stat.exists') | list | unique)
\ No newline at end of file
+  when: not True in (systempython.results | selectattr('stat', 'defined') | map(attribute='stat.exists') | list | unique)
+
+- name: check for python2 on SUSE/openSUSE
+  stat:
+    path: /usr/bin/python
```

Comment:
how do you know whether this is python2 or python3 ?

Reply:
On openSUSE Leap 15.x / SLES 15.x, only python2 provides `/usr/bin/python`.

Reply:
you can remove this task now